bitnami/jenkins/values.yaml

```diff
@@ -444,77 +517,82 @@ metrics:
+  ## @param metrics.resources.limits The resources limits for the Jenkins exporter container
+  ## @param metrics.resources.requests The requested resources for the Jenkins exporter container
   ##
   resources:
-    # We usually recommend not to specify default resources and to leave this as a conscious
-    # choice for the user. This also increases chances charts run on environments with little
-    # resources, such as Minikube. If you do want to specify resources, uncomment the following
-    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
     limits: {}
-    #   cpu: 100m
-    #   memory: 128Mi
     requests: {}
-    #   cpu: 100m
-    #   memory: 128Mi
-
-  ## Metrics exporter service configuration
+  ## Jenkins exporter service parameters
   ##
   service:
-    ## Service type
+    ## @param metrics.service.type Jenkins exporter service type
     ##
     type: ClusterIP
-    ## Metrics exporter port
+    ## @param metrics.service.port Jenkins exporter service port
     ##
     port: 9122
-    ## Specify the nodePort value for the LoadBalancer and NodePort service types.
-    ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#type-nodeport
+    ## @param metrics.service.nodePort Node port for exporter
     ##
     nodePort: ""
-    ## Set the LoadBalancer service type to internal only.
+    ## @param metrics.service.externalTrafficPolicy Jenkins exporter service external traffic policy
+    ## ref: https://kubernetes.io/docs/tasks/access-application-cluster/create-external-load-balancer/#preserving-the-client-source-ip
+    ##
+    externalTrafficPolicy: Cluster
+    ## @param metrics.service.loadBalancerIP Jenkins exporter service Load Balancer IP
     ## ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
     ##
-    # loadBalancerIP:
-    ## Annotations for the Prometheus metrics service
+    loadBalancerIP: ""
+    ## @param metrics.service.loadBalancerSourceRanges Jenkins exporter service Load Balancer sources
+    ## https://kubernetes.io/docs/tasks/access-application-cluster/configure-cloud-provider-firewall/#restrict-access-for-loadbalancer-service
+    ## e.g.
+    ## loadBalancerSourceRanges:
+    ##   - 10.10.10.0/24
+    ##
+    loadBalancerSourceRanges: []
+    ## @param metrics.service.annotations [object] Additional custom annotations for Jenkins exporter service
     ##
     annotations:
       prometheus.io/scrape: "true"
       prometheus.io/port: "{{ .Values.metrics.service.port }}"
-
-  ## Prometheus Operator ServiceMonitor configuration
+  ## Prometheus Service Monitor
+  ## ref: https://github.com/coreos/prometheus-operator
+  ##      https://github.com/coreos/prometheus-operator/blob/master/Documentation/api.md#endpoint
   ##
   serviceMonitor:
+    ## @param metrics.serviceMonitor.enabled Create ServiceMonitor resource(s) for scraping metrics using PrometheusOperator
+    ##
```

Comment:
metrics.serviceMonitor.selector is missing

Reply:
Renaming `metrics.serviceMonitor.selector` to `metrics.serviceMonitor. additionalLabels`